### ISSUE TITLE
Maximum call stack size exceeded in logger

### ISSUE THREAD

**Issue by ramiel** [nextauthjs/next-auth#1783]
**Describe the bug**
I get this error only in version 3.15.1
```
Uncaught RangeError: Maximum call stack size exceeded
    at Object.warn (logger.js:23)
```

when the user is signed in

**Steps to reproduce**
Just login

**Expected behavior**
Should load the page

**Screenshots or error logs**
```
Uncaught RangeError: Maximum call stack size exceeded
    at Object.warn (logger.js:23)
```

**Feedback**
*Documentation refers to searching through [online documentation](https://next-auth.js.org), code comments and issue history. The example project refers to [next-auth-example](https://github.com/iaincollins/next-auth-example).*

* [ ] Found the documentation helpful
* [ ] Found documentation but was incomplete
* [ ] Could not find relevant documentation
* [ ] Found the example project helpful
* [ ] Did not find the example project helpful

**Comment by balazsorban44** [comment]
Could you please provide a reproduction? Your NextAuth config and how do you invoke `signIn`?

**Comment by followbl** [comment]
something is going on with this version...

```tsx
logger.js?3836:15 Uncaught TypeError: code.toLowerCase is not a function
    at Object.error (logger.js?3836:15)
    at console.clientLogger.<computed> [as error] (logger.js?3836:67)
    at index.js:1
    at index.js:1
```

getting this - never gotten anything close to this before.

**Comment by ramiel** [comment]
I have no time to provide a reproduction now but I can tell you that, commenting out the content of the  `warn` function in the `src/lib/logger.js` file, solves the issue.

**Comment by jahirfiquitiva** [comment]
Same thing started happening to me after updating to `3.15.1`

**Comment by balazsorban44** [comment]
Please someone provide a reproduction, otherwise we cannot help.

**Comment by 4ortytwo** [comment]
Getting "too much recursion" error after the latest update. Not sure how to reproduce it.

**Comment by balazsorban44** [comment]
So since several people have reported it, I tried to reproduce it myself, but did not manage. I decided to revisit the last release(s), and found I tiny change, that just seemed very odd, probably introduced by being clumsy, in `3.15.0`. :shrug: so I reverted it, but since I cannot reproduce the issue, could someone in this thread verify that this has been indeed fixed in `3.15.2`?

@ramiel @followbl @jahirfiquitiva @4ortytwo

**Comment by SeanGroff** [comment]
> So since several people have reported it, I tried to reproduce it myself, but did not manage. I decided to revisit the last release(s), and found I tiny change, that just seemed very odd, probably introduced by being clumsy, in `3.15.0`. 🤷 so I reverted it, but since I cannot reproduce the issue, could someone in this thread verify that this has been indeed fixed in `3.15.2` when it is out?
> 
> @ramiel @followbl @jahirfiquitiva @4ortytwo

I can report back as soon as 3.15.2 is released. I'm not seeing 3.15.2 released quite yet.

**Comment by balazsorban44** [comment]
Working out some kind of a release issue, hopefully, will be resolved soon. Thanks for your patience :pray:

**Comment by balazsorban44** [comment]
@SeanGroff it should be out, under `3.15.4`. Could you see if the problem is fixed now?

**Comment by 4ortytwo** [comment]
@balazsorban44 wow, really appreciate this fix and your work. It's back to normal at least for me.

**Comment by SeanGroff** [comment]
> @SeanGroff it should be out, under `3.15.4`. Could anyone see if the problem is fixed now and report back ASAP?

It's working for me as well, thank you so much @balazsorban44 🚀 🚀 🚀

**Comment by balazsorban44** [comment]
:pray: 
For anyone interested, here was the problem :facepalm:, totally unintentional:

![image](https://user-images.githubusercontent.com/18369201/115455055-bfc65a00-a221-11eb-8bd8-e67e43d073a7.png)

**Comment by ramiel** [comment]
Thanks for the quick fix.

**Comment by marciodiasdeveloper** [comment]
solved updated 3.15.10
